Updates to Subscription section (#25041)
docker/docker.github.io: content/manuals/subscription/change.md

When you upgrade your Docker subscription, you immediately get access to all features and entitlements in your new subscription tier.

> [!TIP]

> > If you're upgrading from a free Personal subscription to a Team subscription

Reviewer: **nitpick:** I would drop `free`, otherwise put `paid` in front of Team. But it would sound a bit overloaded `from a free Personal subscription to a paid Team subscription`. 

```suggestion
> If you're upgrading from a Personal subscription to a Team subscription
```

> and want to keep your username, [convert your user account into an organization](../admin/organization/setup/convert-account.md).

To upgrade your subscription:

1. Sign in to [Docker Home](https://app.docker.com/) and select the organization
   you want to upgrade.
1. Select **Billing** to view your current plans.
1. Select **Browse products**.

> 1. Choose **View plans** from the Docker tile on the products catalog page.

Reviewer: **nitpick:** I would highlight that Docker is a product name here. 

<img width="1736" height="795" alt="Image" src="https://github.com/user-attachments/assets/647b9099-0ed3-4615-8b28-73d394615b4a" />

```suggestion
1. Choose **View plans** from the Docker product tile on the products catalog page.
```

> 1. Choose a Docker Team or Business subscription for the organization.

Reviewer: **suggestion:** The user is selecting a plan (Team or Business), there is no subscription at this stage yet.

```suggestion
1. Choose a Docker Team or Docker Business plan for the organization.
```

For feature limits in each tier, see [Docker Pricing](https://www.docker.com/pricing?ref=Docs&refAction=DocsSubscriptionChange).

## Downgrade your subscription

> You can downgrade your Docker subscription at any time before the renewal date. The unused portion isn't refundable, but you retain access to paid features until the next billing cycle.

Reviewer: **suggestion** If the user has downgraded (aka disabled auto-renewal), then there is no next billing cycle.

```suggestion
You can downgrade your Docker subscription at any time before the renewal date. The unused portion isn't refundable, but you retain access to paid features until the end of the current billing cycle.
```
